pyluba/aliyun/cloud_gateway.py

```diff
@@ -50,15 +50,19 @@ class CloudIOTGateway:
     _iotCredentials = {}
     converter = DatatypeConverter()
 
+    _regions_file_path = 'regions.json'
+    _iotCredentials_file_path = 'iotCredentials.json'
+    _mqtt_credentials_file_path = '_mqtt_credentials.json'
+
```

Comment:
I think handling file operations here is the wrong place for it.

This class can have data passed in on init and handle it from there. File operations to store this information need to happen else where.

When it comes to integrations such as HA, all this data will be stored inside a database, so creating files is going to pollute the filesystem, I also think you'll find the files will write to the top level instead of within the project, same issue I ran into with the messages.db.